CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing
+
+This doc explains how to get started contributing to [baikonur-oss](https://github.com/baikonur-oss) project.
+
+## Filing an Issue
+
+If you are trying to use baikonur-oss modules and run into some issues, please file an issue. We can look into the issue more easily if you describe in details as possible as you can. The following is example detailed descriptions:
+
+- Version of tools and modules
+- Expected and actual results
+- Error messages with debug option
+- Steps to reporoduce the issues
+
+## Submitting a PR
+
+Before submitting a PR, make sure that there's an issue filed for your work. It might need some discussion before addressing the issue. Filing a issue first will help us to save your time to solving misunderstandings between you and us.
+
+## Releasing
+
+The release process for existing module goes something like this:
+
+1. Make sure the direction for addressing the issues under discussion
```

Comment:
I could not parse this. Please explain.

Reply:
This is related to the https://github.com/baikonur-oss/docs/pull/1#discussion_r305209300. I think we should discuss how to resolve the issue in filed Issue before accepting a PR.

Reply:
I personally do not think we **must** have issues for PRs at all times.

Can we remove/comment out this line temporarily until we sort this out?